extraConfigBFB.py

```diff
@@ -68,11 +68,24 @@ def ExtraConfigSwitchNicMode(cc: ClustersConfig, _: ExtraConfigArgs, futures: di
 
     client.oc("create -f manifests/nicmode/pool.yaml")
 
+    def helper(h: host.HostWithBF2) -> Optional[host.Result]:
+        h.cold_boot()
+
+    executor = ThreadPoolExecutor(max_workers=len(cc.workers))
     # label nodes
     for e in cc.workers:
         logger.info(client.oc(f'label node {e.name} --overwrite=true feature.node.kubernetes.io/network-sriov.capable=true'))
 
     client.oc("delete -f manifests/nicmode/switch.yaml")
     client.oc("create -f manifests/nicmode/switch.yaml")
+
```

Comment:
Can you add a comment to the jira bug stating this if a WA for the issue (bf-2 firmware failing to update w/o cold boot)

Reply:
Just added it!